### DISCUSSION TITLE
Issue

### DISCUSSION THREAD

**Discussion by Doccaa** [community/community#76514] [GitHub Education]
Issue resolved

**Comment by MegatIrfan** [comment]
Make sure that you are signed in to the correct GitHub account and check if you are part of multiple teams. The GitHub Student Pack promotion may be associated with a specific team, and you need to switch to that team to use the credits.

If you are part of multiple teams, switch to the team to which the GitHub Student Pack promotion was applied. You can switch teams by going to the top-right corner of GitHub, clicking on your profile picture, and selecting the team from the list.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Doccaa , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by Doccaa** [comment]


**Reply by faelperetta** [reply]
how it was solved?

**Reply by SimoElkhair** [reply]
> Issue solved

how do you solve it

**Reply by Lyrilon** [reply]
how do you solve it

**Reply by FahmiKamarul** [reply]
> how do you solve it

so i have actually solve this, I emailed the guy from digital ocean to have it reactivate with the correct perks

**Comment by SimoElkhair** [comment]
> Issue resolved

how do you solve it
